src/js/AccessibilityUtils.js

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 goog.require('axs.browserUtils');
+goog.require('axs.color');
+goog.require('axs.color.Color');
```

Comment:
Ah, thanks for the clarity. I'm happy with whichever approach is more consistent with the style we're using in the project. Up to you.